code/datums/gods/patrons/inhumen_pantheon.dm

```diff
@@ -4,19 +4,8 @@
 	undead_hater = FALSE
 	confess_lines = list(
 		"PSYDON IS THE DEMIURGE!",
-		"THE TEN ARE WORTHLESS COWARDS!",
-		"THE TEN ARE DECEIVERS!",
```

Comment:
Isn't this just an abstract type for others to inherit from? If so it doesn't need lines, if all inheritors define their own.